src/pipecat/transports/smallwebrtc/transport.py

```diff
@@ -939,7 +939,7 @@ async def _on_app_message(self, message: Any):
         """Handle incoming application messages."""
         if self._input:
             await self._input.push_app_message(message)
-        await self._call_event_handler("on_app_message", message)
+        await self._call_event_handler("on_app_message", message, "")
```

Comment:
Since we send the `participant_id` on the `DailyTransport`, maybe here we should send the `peer_id` instead.

In that case, whoever invokes this callback would need to provide it.

You could do something like:
`self._webrtc_connection.pc_id` inside `_handle_app_message`.

This would also require changing the signature of `SmallWebRTCCallbacks`, which is only used internally.

But we can do in a follow up PR to don't keep blocking this one.